src/p_pspr.c

```diff
@@ -111,7 +111,7 @@ void P_SetPspritePtr(player_t *player, pspdef_t *psp, statenum_t stnum)
         }
 
       // killough 7/19/98: Pre-Beta BFG
-      if (stnum == S_BFG1 && (classic_bfg || beta_emulation))
+      if (stnum == S_BFG1 && (STRICTMODE(classic_bfg) || beta_emulation))
```

Comment:
If a demo is recorded with `-complevel mbf` and pre-beta bfg enabled, what happens if strict mode is enabled when playing it back?

Reply:
This should fix the desync (patch for latest master): [prebetabfg.patch](https://github.com/fabiangreffrath/woof/files/14553494/prebetabfg.patch)

Reply:
> If a demo is recorded with `-complevel mbf` and pre-beta bfg enabled, what happens if strict mode is enabled when playing it back?

Realistically, no one has ever recorded a demo with Classic BFG and/or Beta mode 😄 Feel free to commit to this PR if you want to fix this condition.

Reply:
That's fair! 🙂

Reply:
Thanks! 👍